### PR TITLE
Moved non-version-specific content to a new component (eforms-common).

### PR DESCRIPTION
**Pull request by rousso**
Please review and approve but do not merge. I need to make one more change in the nav of version specific branches.

### CODE REVIEW

**Review by gennidou — DISMISSED** [review]

**Review by bertrand-lorentz — APPROVED** [review]
OK as-is, small things in comments.

**Review by ferrakaop — APPROVED** [review]
New structure and pages looks ok.

xref:eforms:schema:index.adoc - the link should read "eForms Schema Specification" to be clearer what it's about.

modules/ROOT/pages/index.adoc - a couple of minor typos could be fixed

File: modules/ROOT/pages/roadmap/index.adoc [hunk 1]
```diff
@@ -9,7 +9,7 @@ This page provides an overview of the upcoming releases and the changes that are
 Since the release of the eForms SDK 1.0.0 in August 2022, we have continued to release updates to the SDK that incrementally improve the eForms metadata that is included in it. These updates are all backwards compatible with the SDK 1.0.0 and therefore they can be used by applications that have already been adapted to the SDK 1.0.0. 
 
 // === SDK 1.12.0
-// SDK 1.12.0, which will be released shortly after SDK 1.11.0, will not include any metadata updates but **will extend the conceptual model** by introducing **Business Entities**  in `fields.json`. 
+// SDK 1.12.0, which will be released shortly after SDK 1.11.0 in March 2024, will not include any metadata updates but **will extend the conceptual model** by introducing **Business Entities**  in `fields.json`.
```
[the remark this answers is not in the document]
Comment (rousso) [reply modules/ROOT/pages/roadmap/index.adoc]: Mistake. Fixed.

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
 build
+node_modeules
```
[the remark this answers is not in the document]
Comment (rousso) [reply .gitignore]: Fixed.

**Comment by rousso** [comment]
@ferrakaop 

> xref:eforms:schema:index.adoc - the link should read "eForms Schema Specification" to be clearer what it's about.

We name the contents of the SDK after the respective folder names.


> modules/ROOT/pages/index.adoc - a couple of minor typos could be fixed

Could not find any typos in that page.

**Pull request merged**